specification/resourcehealth/resource-manager/Microsoft.ResourceHealth/preview/2021-01-01/examples/EmergingIssues_Get.json

```diff
@@ -0,0 +1,84 @@
+{
+  "parameters": {
+    "issueName": "default",
+    "api-version": "2021-01-01"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/providers/Microsoft.ResourceHealth/emergingissues/default",
+        "type": "/providers/Microsoft.ResourceHealth/emergingissues",
```

Comment:
The type should be "Microsoft.ResourceHealth/emergingIssues".  The type is wrong in a number of the examples.  Please go through and clean them up

Reply:
APIs that are not related to potential outages will not be addressed on this PR as this PR is meant to register a new resource type "potential outages". The rest of the spec is consistent with the published stable version. let me know if that's acceptable.

Reply:
These examples are wrong and you shouldn't persist wrong information that customers rely on as API documentation.  They should be fixed now

Reply:
good point, i will fix it in current iteration. thanks